.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: java
+sudo: false
+install: true
```

Comment:
I wonder if this is related to this ticket: https://github.com/travis-ci/travis-ci/issues/3396

Do you see any drawback to comment the `sudo: false` command?